CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing
+
+The following is a set of guidelines for contributing to ERT.
+
+## Ground Rules
+
+1. We use Black code formatting
+1. All code must be testable and unit tested.
+
+## Pull Request Process
+
+1. Work on your own fork of the main repo
+1. Push your commits and make a draft pull request using the pull request template.
+1. Check that your pull request passes all tests.
+1. When all tests have passed and your are happy with your changes, change your pull request to "ready for review"
```

Comment:
You cant really change to ready for review unless you have ZenHub, so this does not apply to outside contributers, so just write that they should create a draft PR and make un-draft when ready for review? A bit awkward that our workflow is slightly different though.

Reply:
![Skjermbilde 2020-12-17 kl  12 03 27](https://user-images.githubusercontent.com/53553193/102479975-f156cb00-405f-11eb-9973-93f78cd5424e.png)

Reply:
I am not logged into zenhub.

Reply:
I was a bit quick and missed you point number 1. (which is in fact number 2).